tests/regressionTests/inputs/slabBurner2D/slabBurner2D.yaml

```diff
@@ -85,15 +85,20 @@ timestepper:
       # class that creates the required fields for the compressible flow solver (rho, rhoE, rhoU, ...)
       - !ablate::finiteVolume::CompressibleFlowFields
         # The soot equation of state is an extension of the tChem eos with support for a simple soot model
-        eos: !ablate::eos::TChemSoot  &eos
-          mechFile: MMAReduced.yml
+        eos: !ablate::eos::TChem  &eos
```

Comment:
Is there no more soot eos?

Reply:
There is, but it is not needed when using the new soot process.